### DISCUSSION TITLE
Unable to Install Chatwoot from Docker Desktop - Switch to Inspect Mode

### DISCUSSION THREAD

**Discussion by afunworm** [chatwoot/chatwoot#8017] [Q&A]
I am using Docker Desktop and pulling chatwoot/chatwoot image. However, running it results in immediate stop of the container with only the following message:

`Switch to inspect mode.`

Is there anything missing here? I've looked up the message and the only thing I have learned so far was that it was related to Rails. How do you fix that from Docker Desktop?

Thank you.

**Comment by twicer-is-coder** [comment]
1 year and no response. I am facing same problem today.

**Comment by sojan-official** [comment]
The base image won't work as it is since it has dependencies, please follow the installation instructions on our docs 
ref: https://github.com/chatwoot/chatwoot/issues/427
